packages/wonder-blocks-popover/components/close-button.js

```diff
@@ -20,6 +20,11 @@ type Props = {|
      * Called when the popover closes
      */
     onClose?: () => mixed,
+
+    /**
+     * Custom styles
+     */
+    style?: StyleType,
```

Comment:
@kevinbarabash As your suggested in PR 1/4, I moved the styles up to the parent container :)